front/src/modules/rollingStock/consts.ts

```diff
@@ -212,6 +218,12 @@ export const RS_SCHEMA_PROPERTIES: readonly SchemaProperty[] = [
     side: 'middle',
     condition: isElectric,
   },
+  {
+    title: 'supportedSignalingSystems',
+    type: 'checkboxes',
+    enum: ['BAL', 'BAPR', 'TVM300', 'TVM430'],
```

Comment:
This enum should not be statically defined in the front. You have an endpoint to retrieve the list of signaling systems the application supports. Check out this endpoint  `/sprites/signaling_systems`.